README.md

```diff
@@ -10,7 +10,7 @@ Community (college) maintained list of algorithm implementations and codes.
 
 | Algorithm | C | CPP | Java | Python |
 |:--------------|:----------------:|:----------------:|:----------------:|:-----------------:|
-| [Binary Search](https://en.wikipedia.org/wiki/Binary_search_algorithm) | [\[X\]](binary_search/binary_search.c) ||||
+| [Binary Search](https://en.wikipedia.org/wiki/Binary_search_algorithm) | [\[X\]](binary_search/binary_search.c) | [\[X\]](binary_search/BinarySearch.java) |||
```

Comment:
You have mistakenly added the file's link in the CPP column, Please add this in JAVA column